src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,57 @@
+package bit.javaoop;
+
+public class CEO {
+    private final int size = 10;
+    private Manager tab[] = new Manager[size];
+    private int counter;
+    private String name;
+
+    public CEO(String name) {
+        this.name = name;
+    }
+
+    public int getSize() {
+        return size;
+    }
+
+    public Manager[] getTab() {
+        return tab;
+    }
+
+    public void setTab(Manager[] tab) {
+        this.tab = tab;
+    }
+
+    public int getCounter() {
+        return counter;
+    }
+
+    public void setCounter(int counter) {
+        this.counter = counter;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void addManager(Manager manager){
+        if(counter < size)
+            tab[counter++] = manager;
+    }
+    @Override
+    public String toString(){
+        String text = "";
+        text+=getName() + " - CEO\n" ;
+        for(int i = 0; i < getCounter(); i++){
+            text+="\t"+getTab()[i].getName() + " - " + getTab()[i].getClass().getSimpleName() +"\n"; /// getClass zwraca "klase", getSimpleName zwraca nazwe klasy
```

Comment:
- Takie konkatenowanie w pętlach lepiej zrobić za pomocą StringBuildera nawet IDE podpowiada po najechaniu na podświetlenie i wciśnięciu alt + enter 
![image](https://user-images.githubusercontent.com/18233796/37979172-2cc4dc60-31e8-11e8-907b-073ce70fdf02.png)
Ogólnie intellij jest całkiem mądrym narzędziem i jak coś podświetla to bardzo często chce powiedzieć/zaproponować coś mądrego ^_^

- Nie używamy .getClass().getSimpleName() to jest takie już troszkę hakerskie sięganie do bebechów klasy bardzo nie obiektowo lepiej byłoby dodać w interfejsie Managera np. `public string getName()` i później sobie w konkretnych implementacjach tej metodyt zwracać return "FixedBudgetManager". 

Bo w tym momencie tak tworzymy interfejs czyli idziemy z abstrakcją w górę, a później hakersko odwołujemy do nazwy konkretnej implementacji tego interfejsu. 

Po to tworzymy abstrakcję żeby nie przejmować się konkretną implementacją a tutaj sięgasz do konkretnej implementacji.